pwiz_tools/Skyline/Menus/ViewMenu.resx

```diff
@@ -327,6 +327,18 @@
   <data name="zMenuItem.Text" xml:space="preserve">
     <value>&amp;Z</value>
   </data>
+  <data name="zhMenuItem.Size" type="System.Drawing.Size, System.Drawing">
+    <value>130, 22</value>
+  </data>
+  <data name="zhMenuItem.Text" xml:space="preserve">
+    <value>&amp;Z</value>
+  </data>
+  <data name="zhhMenuItem.Size" type="System.Drawing.Size, System.Drawing">
+    <value>130, 22</value>
+  </data>
+  <data name="zhhMenuItem.Text" xml:space="preserve">
```

Comment:
Looks like a copy/paste glitch

Reply:
It's not clear what shortcut to use in this case, since there are no typeable characters there except 'Z'.

Reply:
I would make the text Z+1 and Z+2 and make the 1 and 2 the mnemonics.